src/pages/informational/WpPlugin.tsx

```diff
@@ -0,0 +1,225 @@
+import character from "assets/laira/laira-waiving.png";
+import Copier from "components/Copier";
+import Image from "components/Image";
+
+export default function WpPlugin() {
+  return (
+    <div
+      className={`padded-container mt-16 grid content-start pb-16 @container`}
+    >
+      <h1 className="text-center @6xl:text-left text-4xl @6xl:text-5xl @6xl:leading-tight text-pretty mb-4 text-navy-d4">
+        Better Giving Wordpress plugin documentation
+      </h1>
+      <div className="py-5">
+        <h2 className="mb-4 col-span-full text-lg text-blue-d1 uppercase text-center">
+          Installing the WordPress Better Giving Donation Form Plugin
+        </h2>
+        <ol className="list-decimal">
+          <li>
+            Download the Better Giving{" "}
+            <a
```

Comment:
can use `components/ExtLink` so no longer need to set `target=_blank` and `rel`
```suggestion
            <ExtLink
```